code/modules/projectiles/guns/projectile/sniper.dm

```diff
@@ -148,3 +145,74 @@
 	dismemberment = 0
 	weaken = 0
 	breakthings = FALSE
+
+//The Dragon
+
+/obj/item/weapon/gun/projectile/automatic/sniper_rifle/dragon
+	name = "dragon"
+	desc = "cheap and effective, what more could you want from a DMR"
```

Comment:
Capitalise the start of the sentence and full stop the end, like the other descriptions